packages/components/drawer/src/drawer.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <teleport to="body" :disabled="!appendToBody">
+  <el-teleport to="body" :disabled="!appendToBody">
```

Comment:
Maybe we can make `to` here configurable later?

Reply:
Agree, It should be consistent with the ElDialog